src/pages/ControllersIndex/ControllersIndex.tsx

```diff
@@ -129,25 +120,50 @@ function Details() {
     const region =
       "location" in c && c?.location?.region ? c.location.region : "unknown";
     const cloudRegion = `${cloud}/${region}`;
-    const publicAccess = "Public" in c && c.Public ? `${c?.Public}` : "False";
-
+    const access = "Public" in c && c.Public ? "Public" : "Private";
+    let columns = [
+      generatePathValue(c),
+      { content: cloudRegion },
+      { content: c.models },
+      { content: c.machines },
+      { content: c.applications },
+      { content: c.units },
+      { content: access, className: "u-capitalise" },
+    ];
+    if ("version" in c && c.version) {
+      // insert before the last column
+      columns.splice(columns.length - 1, 0, {
```

Comment:
This column needs to be set even if there isn't a version otherwise there will be a mis-match between the number of headers and columns so the access value will appear under the version column.

<img width="1263" alt="Screenshot 2023-03-14 at 9 19 36 am" src="https://user-images.githubusercontent.com/361637/224845782-c146f108-5617-497e-aa14-f5637bc358ff.png">

Reply:
Good point!